packages/extension-vscode/src/extension.ts

```diff
@@ -7,7 +7,11 @@ import {
     TransportKind
 } from 'vscode-languageclient';
 
-import * as notifications from './notifications';
+import * as notifications from './utils/notifications';
+
+type TelemetryState = 'ask' | 'disabled' | 'enabled';
```

Comment:
You are using this in at least a couple of files, maybe it is better to move this to a place to share it.